projects/uitdatabank/docs/entry-api/reuse-places-organizers.md

```diff
@@ -1,304 +1,10 @@
-
-* ✅ Suitable when the organizers for the events created through your application are unpredictable in advance
-* ❌ Not recommended when the events created through your application all have the same organizer
-* ❌ Not recommended when the events created through your application can have a different organizer, but (1) the amount of organizers is limited & (2) the list of organizers is static and won't change a lot over time
-
-In this solution you save the relevant data for organizers inside your own application / database. Before exporting the event data to UiTdatabank a check is made to see if the organizers linked to the events already exist in UiTdatabank or not.
-
-This check can be done on the following fields:
-
-* `name`
-* `website` (this is used to keep organizers unique in UiTdatabank)
-
-<!-- theme: warning -->
-
-> While the `name` filter works for a lot of cases, it is fragile because it depends on the name being spelled exactly the same in your application as in UiTdatabank. For example, "Koninklijk Museum voor Schone Kunsten Antwerpen" is often abbreviated to "KSMKA" or "K.M.S.K.A" by users who enter event data.
->
-> For this reason we advice to always filter on `website` when possible as this is less often spelled differently and we take variations like trailing `/` and no trailing `/` or `www.` or no `www.` into account. If you do not know the website of the organizer, filter on `name` but provide a way for your users to [select the right UiTdatabank organizer in your own entry form](#searching-existing-organizers-within-your-entry-form) when possible because this way the user can change the spelling of the name to other variations to find the right match.
-
-#### Example
-
-**request**
-
-In the request below the check is being made on the `website` of the organizer
-
-```
-GET /organizers/?website=https://www.natuurpunt.be/afdelingen/natuurpunt-holsbeek
-```
-
-**response**
-
-```js
-{
-   "@context":"http://www.w3.org/ns/hydra/context.jsonld",
-   "@type":"PagedCollection",
-   "itemsPerPage":30,
-   "totalItems":1,
-   "member":[
-      {
-         "@id":"https://io.uitdatabank.be/organizers/e83edb7f-6f23-47cd-a730-66fd7687f116",
-         "@type":"Organizer"
-      }
-   ]
-}
-```
-
-If the organizer already exists (like in the example above), the URL of that organizer (`https://io.uitdatabank.be/organizers/e83edb7f-6f23-47cd-a730-66fd7687f116` in this case) must be used as `organizer.@id` in the `POST /events` request of the event.
-
-Only when no results are found a new organizer can be created in UiTdatabank. When doing so, the UiTdatabank API will return a response containing the URL of the organizer. This organizer URL must be used then in the `POST /events` request of the event.
+<!-- 
+  This page is not listed in the sidebar, but we should keep it for anyone that has the 
```

Comment:
I don't think this link is already used by so many external parties? Maybe just link to the new guide?

Reply:
I've linked to it already a couple of weeks ago in an email to an integrator that had questions about this

Reply:
And it's easy to have a "forward" like this I think? Also a good example for future guides that get split up. (For simple moves we can do a redirect in Stoplight)

Reply:
But now we introduce a forward page because 1 email is sent to an integrator?
Also every new integrator that is now on the "Requirements before going live" page, has to click twice to go to the correct reuse guide: one time on the link in the "Requirements before going live" page and one time on the "forward page"

Reply:
Yes I just noticed the "Requirements before going live" page, I overlooked that. Of course we should link to the separate guides there.

For the link in the email: I just think it's best practice in general, a good example for anyone else who does something like this in the future when the docs are more heavily used, and it's not hard? It's already in place like this and no one will unnecessarily see it when I update the links in our other guides to link to the separate guides

Reply:
I've fixed the links in other guides to the old page in https://github.com/cultuurnet/apidocs/pull/134/commits/f1d380f2746db1ff293ac3da8e4bbefc38eb77dc

Reply:
Okay, thanks 👍 
I have nothing against a forward page, but with the link in the "Requirements before going live" page to the forward page it felt like the solution for one integrator (with link) introduced a problem for all other integrators (without link)

Reply:
Yep I understand now, that makes sense but an oversight on my end :)